### PR TITLE
try setting MAX_JOBS=4 for oom in arm wheel

### PR DESCRIPTION
**Pull request by tinglvv**
https://github.com/pytorch/pytorch/actions/runs/8840652730/job/24276381274?pr=124112 hitting OOM error in building cuda ARM wheel. 
Try changing MAX_JOBS.

### CODE REVIEW

**Review by nWEIdia — APPROVED** [review]
Looks great!

File: aarch64_linux/aarch64_wheel_ci_build.py [hunk 1]
```diff
@@ -122,8 +122,6 @@ def update_wheel(wheel_path) -> None:
         "/usr/local/cuda/lib64/libcudnn_cnn_train.so.8",
         "/usr/local/cuda/lib64/libcudnn_ops_infer.so.8",
         "/usr/local/cuda/lib64/libcudnn_ops_train.so.8",
-        "/opt/conda/envs/aarch64_env/lib/libopenblas.so.0",
```
Comment (nWEIdia) [inline aarch64_linux/aarch64_wheel_ci_build.py:125]: This was deleted. Is there another place that we can put this libopenblas.so.0?

**Review by nWEIdia — COMMENTED** [review]
Re-reviewing since we are having libopenblas.so test issues.

File: aarch64_linux/aarch64_wheel_ci_build.py (hunk 1 above)
Comment (tinglvv) [reply aarch64_linux/aarch64_wheel_ci_build.py:125]: right, now we should copy from /opt/OpenBLAS

**Comment by nWEIdia** [comment]
Please rebase so that the s390x errors will not show up:  https://hud.pytorch.org/pytorch/pytorch/pull/126174 

For [the cuda test failure](https://github.com/pytorch/pytorch/actions/runs/9138780123/job/25133988980)s, we need to wait for ARM + CUDA instance availability: e.g. https://aws.amazon.com/ec2/instance-types/g5g/ 

cc @atalman @malfet @ptrblck

**Review by nWEIdia — CHANGES_REQUESTED** [review]
Rebase is needed to fix some ibm errors. 
Otherwise, looks great!

**Comment by tinglvv** [comment]
> Please rebase so that the s390x errors will not show up: https://hud.pytorch.org/pytorch/pytorch/pull/126174
> 
> For [the cuda test failure](https://github.com/pytorch/pytorch/actions/runs/9138780123/job/25133988980)s, we need to wait for ARM + CUDA instance availability: e.g. https://aws.amazon.com/ec2/instance-types/g5g/
> 
> cc @atalman @malfet @ptrblck

Thanks for reviewing. I think we need the SBSA nvidia driver 550.54.15 to be uploaded to AWS instead of the instance availability. I started https://github.com/pytorch/test-infra/pull/5218/files to be merged once we upload the sbsa nvidia driver runfile to https://s3.amazonaws.com/ossci-linux/nvidia_driver/.

**Comment by nWEIdia** [comment]
> > Please rebase so that the s390x errors will not show up: https://hud.pytorch.org/pytorch/pytorch/pull/126174
> > For [the cuda test failure](https://github.com/pytorch/pytorch/actions/runs/9138780123/job/25133988980)s, we need to wait for ARM + CUDA instance availability: e.g. https://aws.amazon.com/ec2/instance-types/g5g/
> > cc @atalman @malfet @ptrblck
> 
> Thanks for reviewing. I think we need the SBSA nvidia driver 550.54.15 to be uploaded to AWS instead of the instance availability. I started https://github.com/pytorch/test-infra/pull/5218/files to be merged once we upload the sbsa nvidia driver runfile to https://s3.amazonaws.com/ossci-linux/nvidia_driver/.

The error message was "RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx" 

Would we need to have a nvidia gpu first and then installing a driver? The M7G instance does not have an NVIDIA GPU.

**Comment by nWEIdia** [comment]
And good catch, eventually we would need SBSA nvidia driver 550.54.15 to be uploaded to AWS for the test to work.

**Comment by nWEIdia** [comment]
The rebase command may not work on pytorch/builder repo. A manual rebase is needed.

**Comment by tinglvv** [comment]
Yes indeed we will need an ARM+CUDA instance, thanks for catching that.
Current instance-type: t4g.2xlarge, which is X86+CUDA based. We can switch once the proper builder is available.

**Comment by tinglvv** [comment]
please ignore the above commits created by rebase, will resolve these later.

**Pull request closed without merging**